global/functions/spawning/fn_spawning_find_valid_position_tracer.sqf

```diff
@@ -46,28 +73,55 @@ private _index = 0;
 while {true} do {
 	_index = _index + 1;
 
+	// find the next point on the line
 	private _stepSize = ((_tracerPosition distance2D _tracerEndPos) / 10) max 100;
-	_lastTracerPosition = _tracerPosition;
-	_tracerPosition = _tracerPosition getPos [_stepSize, _attackDir];
-
-	//Places debug markers on the map when tracers are fired.
-	if (!isNil "debugAttackTracer") then {
-		private _mark = createMarker ["Tracer" + str diag_tickTime + str _index, _tracerPosition];
-		_mark setMarkerType "mil_dot";
-		_mark setMarkerColor "ColorPink";
-		tracerMarkers pushBack _mark;
+	private _newTracerPosition = _tracerPosition getPos [_stepSize, _attackDir];
```

Comment:
Note to self -- this means the `_tracerStart` is never checked as a possible valid point on the line projection.